### DISCUSSION TITLE
How to follow up cyber attacks when it occurred?

### DISCUSSION THREAD

**Discussion by sjgknight** [sjgknight/DSI_collection_sheets#19] [Q&A] [answered]
There are uncountable cyber attacks although cyber secure methods developments. However, we cannot find many news about cyber attacker arrested, therefore, the range of  damage from cyber attacks have not been confirmed. Hackers keep trying invade advanced secure technologies. Accordingly, I believe that following up the attacks have to be preceded. If enterprises lost much customers' information, what can they do for the victims? For example, my passport and contact information in airline, which I frequently use, had been stolen by data thieves. In spite of the situation, they did not guarantee anything and I. changed to different airline. From the reason, I disappointed with them. Through this reason, I would like to know the active solutions for the future, when I experience in these situations as a data scientist.

**Comment by passiona** [comment, marked as answer]
It is best practice for corporations to have data breach plans, which detail the steps they follow in the event of a data breach. As a data scientist get acquainted with your employer's data breach plan or assist with putting one together if they do not have one. The Office of the Australian Information Commissioner (OAIC) has a guide on what makes an effective data breach plan (https://www.oaic.gov.au/privacy/guidance-and-advice/data-breach-preparation-and-response).  This includes containing the breach or making sure it has been stopped and is not continuing. The breach should then be assessed, to determine what information has been breached and when. Depending on what has been breached the next step is to notify those affected. Under Australia's mandatory data breach reporting, if the information breached is likely to cause serious harm, then the OAIC and those affected individuals must be notified. Finally, the company should review the event in order to improve their system,s and processes for the future.

Additional steps that we have seen in recent events that have assisted individuals affected by the breach include, being very specific about the information that has been breached, for example, the exact identification verification document stolen (such a passport or licence) that allows individuals to cancel old ones and obtain new ones. In the Optus breach, Optus also offered to pay for a credit monitoring service for affected customers for a year, to assist with minimising fraudulent use of stolen identifications to access credit or finances. A corporation might even voluntarily offer to pay for the replacement of identification documents that have been breached to assist customers.
